### ISSUE TITLE
Package minted Error

### ISSUE THREAD

**Issue by eltos** [aergus/dockerfiles#6]
After your recent update on 28 Jan 2019 (d91a232253111648a716c1c54dc92bb88719e705 and 1aa8fbe785ed409326f945bfa009247858f2c4a8) I get the following following error when compiling a latex file that used the `minted` package for syntax highlighting:
```
Package minted Error: You must have `pygmentize' installed to use this package.
```
Even though [`python-pygments` is in the Dockerfile](https://github.com/aergus/dockerfiles/blob/1aa8fbe785ed409326f945bfa009247858f2c4a8/latex/Dockerfile#L33).

-------------

**Expected**  
Observed with the previous build before the change (which was cached on my local machine)  
Image sha256 `83cb44d6647e`
```bash
$ which pygmentize
/usr/bin/pygmentize
```

**Observed**
With the new build
Image sha256 `ec713c2b1249`
```bash
$ which pygmentize
ERROR: Job failed: exit code 1
```

**Comment by eltos** [comment]
Using the plain docker image `debian:testing` I ran the following commands (note the python version):
```
$ apt-get update && apt-get install -y python-pygments
$ which pygmentize; echo $?
1
```
```
$ apt-get update && apt-get install -y python3-pygments
$ which pygmentize; echo $?
/usr/bin/pygmentize
0
```
```
$ apt-get update && apt-get install -y python-pip
$ pip install Pygments
$ which pygmentize; echo $?
/usr/local/bin/pygmentize
0
```

**Comment by eltos** [comment]
Just noticed that this issue is already on [gitlab](https://gitlab.com/aergus/dockerfiles/issues/2)

**Comment by aergus** [comment]
This should be fixed by #5.
